<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>450</width>
      <height>370</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Input</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>420</width>
        <height>190</height>
       </rect>
      </property>
      <property name="title">
       <string>Open file</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>381</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="files_selectro_Layout">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>5</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="1" column="2">
         <widget class="QPushButton" name="OpenSP_btn">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="SP_line"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="SP_label">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>Spectrum Folder</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>381</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <widget class="QRadioButton" name="raw_rdbtn">
          <property name="text">
           <string>*.raw</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QRadioButton" name="mzML_rdbtn">
          <property name="text">
           <string>*.mzML</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="csv_rdbtn">
          <property name="text">
           <string>*.csv</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="FileExtension_Label">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>File extension</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="Next1_btn">
      <property name="geometry">
       <rect>
        <x>355</x>
        <y>210</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Noise Filter</string>
     </attribute>
     <widget class="QGroupBox" name="gBox_Filter">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>420</width>
        <height>190</height>
       </rect>
      </property>
      <property name="title">
       <string>Filter</string>
      </property>
      <widget class="QCheckBox" name="EnableFilter_cbtn">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable Filtering</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="gBox_NoiseFilter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>401</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>381</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="2">
          <widget class="QSpinBox" name="BreakCount_spinbox">
           <property name="maximum">
            <number>300</number>
           </property>
           <property name="value">
            <number>200</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Break Count</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="IntensePerc_spinbox">
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="value">
            <double>0.800000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Max Intensity Percentile</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="Next2_btn">
      <property name="geometry">
       <rect>
        <x>355</x>
        <y>210</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Blank</string>
     </attribute>
     <widget class="QGroupBox" name="gBox_Blank_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>420</width>
        <height>190</height>
       </rect>
      </property>
      <property name="title">
       <string>Blank Subtraction</string>
      </property>
      <widget class="QCheckBox" name="EnableBlank_cbtn">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable Blank Subtraction </string>
       </property>
      </widget>
      <widget class="QGroupBox" name="gBox_BlankSubtract">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>401</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QWidget" name="gridLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>381</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="files_selectro_Layout_2">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>5</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="1" column="2">
          <widget class="QPushButton" name="OpenBL_btn">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="BL_line"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="BL_label">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text">
            <string>Blank Spectra</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="Next3_btn">
      <property name="geometry">
       <rect>
        <x>355</x>
        <y>210</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Find method</string>
     </attribute>
     <widget class="QGroupBox" name="gBox_SearchMethod">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>420</width>
        <height>190</height>
       </rect>
      </property>
      <property name="title">
       <string>Search method</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="EC_Dialog_btn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>150</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>Elemental Composition</string>
       </property>
      </widget>
      <widget class="QPushButton" name="DB_Dialog_btn">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>60</y>
         <width>150</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>Compounds List</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
